foundation/foundation-drift-detection/aissemble-foundation-drift-detection-client-python/src/drift/detection/rest/client/drift_rest_client.py

```diff
@@ -68,10 +68,10 @@ def invoke_drift(self, policy_identifier, input, control):
         query_params = {"policyIdentifier": policy_identifier}
         data_headers = {"Content-type": "application/json"}
         # uncomment below for debugging purposes
-        # print('JSON being sent to drift invocation service: ' + drift_data_input.json())
+        # print('JSON being sent to drift invocation service: ' + drift_data_input.json(serialize_as_any=True))
```

Comment:
This might be the way its appearing in the GH UI, but in the off chance that its not, can you revert this such that it one line or adjust it so the indentations are in line with the rest of the code.

Reply:
We should really just remove this or use an actual logger and output to debug and uncomment it.

Reply:
+1 on Emily. 
I feel like we can just remove it and have devs just add this in their testing branch if they are debugging as I don't see people put debugging log unless anyone have strong opinion to keep it.